aip/general/0154.md

```diff
@@ -54,6 +54,32 @@ message Book {
 **Note:** ETag values **should** include quotes as described in [RFC 7232][].
 For example, a valid etag is `"foo"`, not `foo`.
 
+### Etags on request methods
+
+In some situations, the etag needs to belong on a request message rather than
+the resource itself. For example, an `Update` standard method can "piggyback"
+off of the `etag` field on the resource, but the `Delete` standard method can
```

Comment:
```suggestion
off the `etag` field on the resource, but the `Delete` standard method can
```

I have a visceral reaction to "off of" in pretty much any context. Would just "off" be okay here?